README.md

```
@@ -14,11 +14,11 @@
 >
 > We'd love for you to try it out and share your feedback! Please [open an issue](https://github.com/auth0/Auth0.swift/issues) if you encounter any problems or have suggestions.
 >
-> 📚 [Migration Guide](https://github.com/auth0/Auth0.swift/blob/3.0.0-beta.2/V3_MIGRATION_GUIDE.md) &nbsp;•&nbsp; 📦 [v3 Changelog](https://github.com/auth0/Auth0.swift/blob/3.0.0-beta.2/CHANGELOG.md) &nbsp;•&nbsp; 🤖 [Migration Skill](https://github.com/auth0/agent-skills/tree/main/plugins/auth0/skills/auth0-swift-major-migration)
+> 📚 [Migration Guide](https://github.com/auth0/Auth0.swift/blob/3.0.0-beta.2/V3_MIGRATION_GUIDE.md) &nbsp;•&nbsp; 📦 [v3 Changelog](https://github.com/auth0/Auth0.swift/blob/3.0.0-beta.2/CHANGELOG.md) &nbsp;•&nbsp; 🤖 [Auth0 Skill](https://github.com/auth0/agent-skills)
 >
-> **Skill for Coding Agents:** If you use coding agents such as Claude Code or Cursor, add the Auth0.swift migration skill to automate the upgrade:
+> **Skill for Coding Agents:** If you use coding agents such as Claude Code or Cursor, add the Auth0 skill to automate the upgrade:
 > ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)